### PR TITLE
Upgrade v1 rbln sampler

### PR DESCRIPTION
**Pull request by rebel-jongho**
### 🚀 Pull Request Summary

Add RBLN-optimized top_p sampler to v1 engine.

---

### 📌 Related Issues / Tickets

* Fixes #
* Related to # https://github.com/rebellions-sw/vllm-rbln/issues/18

---

### ✅ Changes

* [x] Feature
* [ ] Bug Fix
* [ ] Documentation
* [ ] Refactor
* [ ] Optimization
* [ ] Other (please describe):

---

### 📸 Screenshots / Logs (if applicable)

---

### 🧪 How to Test

1. Run `...`
2. Verify output: `...`
3. Edge case tested: `...`

---

### 📋 Checklist

* [ ] I’ve added unit tests or updated existing ones.
* [ ] I’ve updated relevant documentation.
* [ ] I’ve checked compatibility (e.g., hardware/backend/platform).

---

### 💬 Notes

---

### CODE REVIEW

File: vllm_rbln/v1/worker/optimum_model_runner.py [hunk 1]
```diff
@@ -97,7 +109,16 @@ def __init__(self, vllm_config: VllmConfig, device: torch.device):
...
+        use_rbln_sampler = _use_rbln_sampler()
+        logger.info("Using RBLN sampler: %s", use_rbln_sampler)
+
+        sampler = RBLNSampler() if use_rbln_sampler else Sampler()
+
+        if use_rbln_sampler:
+            # Use torch.compile for optimized RBLN sampler
+            sampler = torch.compile(sampler, dynamic=False, fullgraph=False)
+
+        self.sampler = sampler
```
Comment (rebel-jonghewk) [inline vllm_rbln/v1/worker/optimum_model_runner.py:121]: FYI: This change sets the default behavior to eager mode instead of the torch.compile (CPU) version.

**Review by rebel-jonghewk — APPROVED** [review]
